manifests/collectd_utils.pp

```diff
@@ -0,0 +1,64 @@
+#
+define install_collectd::collectd_utils (
+    $plugin_conf_dir,
+    $config_file,
+    $purge        = undef,
+    $recurse      = undef,
+    $purge_config = false
+) {
+    
+    $typesdb                = []
+    $include                = []
+    $interval               = 10
+    $threads                = 5
+    $timeout                = 2
+    $fqdnlookup             = true
+    $collectd_hostname      = $::hostname
+    $write_queue_limit_high = undef
+    $write_queue_limit_low  = undef
+    file { 'collectd.d':
```

Comment:
Move and put a markerfile in our collectd install so that we don't keep moving out of the way